java/code/src/com/suse/manager/webui/Router.java

```diff
@@ -324,6 +325,13 @@ public void init() {
                 withProductAdmin(ProductsController::synchronizeSubscriptions));
         post("/manager/admin/setup/sync/repositories",
                 withProductAdmin(ProductsController::synchronizeRepositories));
+
+        // Admin - monitoring
+        get("/manager/admin/config/monitoring",
```

Comment:
To reduce the complexity of the router.java and stop the exponential growth of this file it would be nice to have a more modular approach for this. I would suggest following the structure we are already using in the last features: 

Router: https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/suse/manager/webui/Router.java#L98

Controller: https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/suse/manager/webui/controllers/contentmanagement/ContentManagementViewsController.java#L66